docs/sql-keywords.md

```diff
@@ -57,6 +57,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>CASCADE</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>CASE</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>CAST</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
+  <tr><td>CATALOG</td><td>reserved</td><td>reserved</td><td>non-reserved</td></tr>
```

Comment:
This is non-reserved in pgsql and sql 2011, please see https://www.postgresql.org/docs/current/sql-keywords-appendix.html